src/FileIndexer.ts

```diff
@@ -305,7 +322,7 @@ export class FileIndexer {
     return undefined
   }
 
-  private scipSymbol(node: ts.Node): ScipSymbol {
+  private scipSymbol = (node: ts.Node): ScipSymbol => {
```

Comment:
nit: Superfluous change?

Reply:
This change is required for a call in the `uniqueDeclarationSymbolValues` to bind the `scipSymbol` method to the `FileIndexer` instance.

Reply:
I'm confused...  other methods don't use this declaration style, and they can be called just fine... Is it because of the lambda in the `uniqueDeclarationSymbolValues` method?